### ISSUE TITLE
manual/detailedTopics/production/Deploying-CleverCloud.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#909]
- https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/detailedTopics/production/Deploying-CleverCloud.md
- diffs between [2.2.0/manual/detailedTopics/production/Deploying-CleverCloud.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/detailedTopics/production/Deploying-CleverCloud.md) and [2.3.5/manual/detailedTopics/production/Deploying-CleverCloud.md](https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/detailedTopics/production/Deploying-CleverCloud.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/detailedTopics/production/Deploying-CleverCloud.md  2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/detailedTopics/production/Deploying-CleverCloud.md 2014-07-28 12:26:38.000000000 +0900
@@ -1,5 +1,6 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 # Deploying to Clever Cloud
-[Clever Cloud](http://www.clever-cloud.com/en) is a Platform as a Service solution. You can deploy on it Scala, Java, PHP, Python and Node.js applications. Its main particularity is that it supports **automatic vertical and horizontal scaling**.
+[Clever Cloud](https://www.clever-cloud.com/en/) is a Platform as a Service solution. You can deploy on it Scala, Java, PHP, Python and Node.js applications. Its main particularity is that it supports **automatic vertical and horizontal scaling**.

 Clever Cloud supports Play! 2 applications natively. The present guide explains how to deploy your application on Clever Cloud.


```

**Comment by garbagetown** [comment]
:hand:

**Comment by garbagetown** [comment]
merged at #1110
